config/routes.rb

```diff
@@ -2,6 +2,7 @@
   devise_for :users
 
   root 'static_pages#home'
+  get 'about' => 'static_pages#about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.